deploy.py

```diff
@@ -56,34 +86,91 @@ def allow_tcp(port, from_ip="0.0.0.0/0"):
 
 
 class CloudFormationBuilder(object):
-    def __init__(self):
+    def __init__(self, use_firehose=False, use_processor=False):
         self._random_id = str(uuid.uuid4()).replace('-', '')[:12].upper()
         self._template = Template()
         self._template.add_version("2010-09-09")
+        desc = "AWS CloudFormation Push Stack"
+        if use_processor:
+            desc += " - with Firehose Logging + Processor + Push Messages API"
+        elif use_firehose:
+            desc += " - with Firehose Logging"
+        self._template.add_description(desc)
+        self.use_firehose = use_firehose or use_processor
+        self.use_processor = use_processor
         self.add_resource = self._template.add_resource
         self.add_parameter = self._template.add_parameter
 
         self.AutopushVersion = self.add_parameter(Parameter(
             "AutopushVersion",
             Type="String",
             Description="Autopush version to deploy",
-            Default="1.14.1",
+            Default="1.14.2",
             AllowedValues=[
+                "latest",
+                "1.14.2",
                 "1.14.1",
             ]
         ))
-        self.PushVPC = self.add_parameter(Parameter(
-            "ExistingVPC",
-            Type="AWS::EC2::VPC::Id",
-            Description=(
-                "The VPC Id so launch the Autopush server into"
+
+        if self.use_processor:
+            self.ProcessorLambdaBucket = self.add_parameter(Parameter(
+                "ProcessorLambdaBucket",
+                Type="String",
+                Default="push-lambda-funcs",
+                Description="S3 Bucket of lambda Message Processor",
```

Comment:
Lambda is capitalized elsewhere when referred to in docs. Should this be capitalized?